compose-e2e-test.yml

```diff
@@ -0,0 +1,37 @@
+# compose for running e2e tests in CI
+version: "2"
+
+services:
+  remark42:
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        - BUILDPLATFORM=linux/amd64
+        - SKIP_BACKEND_TEST=true
+        - SKIP_FRONTEND_TEST=true
+
+    image: umputun/remark42:dev
+    container_name: "remark42"
+
+    environment:
+      - REMARK_URL=http://remark42:8080
+      - SECRET=12345
+      - DEBUG=true
+      - ADMIN_PASSWD=password
+      - AUTH_DEV=true # activate local oauth "dev"
+      - ADMIN_SHARED_ID=dev_user # set admin flag for default user on local oauth2
+      - POSITIVE_SCORE=false # restricts comment's score to be only positive
```

Comment:
False by default, no point adding it here.

Reply:
Then it should be fixed on dev composes.

Reply:
Good point. Feel free to do so: it indeed doesn't make sense. I assume some frontend developer was playing with non-default values of these options and then didn't exclude them from the commit: 5809419bcece68350d55e4fe95076edfcf917250.